src/libraries/System.Private.DataContractSerialization/src/System/Text/Base64Encoding.cs

```diff
@@ -53,10 +53,8 @@ private bool IsValidTailBytes(int v3, int v4)
             return !(v3 == 64 && v4 != 64);
         }
 
-        public unsafe override int GetByteCount(char[] chars, int index, int count)
+        public unsafe override int GetByteCount(char[] chars!!, int index, int count)
         {
-            if (chars == null)
-                throw System.Runtime.Serialization.DiagnosticUtility.ExceptionUtility.ThrowHelperError(new ArgumentNullException(nameof(chars)));
             if (index < 0)
```

Comment:
This method switched to `!!`, but the others are ArgumentNullException.ThrowIfNull even when they could use `!!`.

I think I've decided your style here was "all or nothing", and since this one hit "all" it's OK, and the rest hit interrupted flow so you went with "nothing".

That's fine, it just struck me as "different" so I thought I'd highlight it in case you felt differently about it now that time has passed.

Reply:
> I think I've decided your style here was "all or nothing"

Yup, that seemed like the cleanest approach per method.  I'm ok if the owners of the code subsequently decide they want to switch to a different style.  I'd also be ok if the owners decided that many of these cases deserve a small breaking change to throw the null exceptions first and thus consistently use !! everywhere.  I just didn't want to make such changes as part of this initial wave.